docs/source/tutorials/building_blocks.rst

```diff
@@ -0,0 +1,247 @@
+#######################
+Manim's building blocks
+#######################
+
+This document explains the building blocks of manim and will give you all the
+necessary tools to start producing your own videos.
+
+Essentially, manim puts at your disposal three different concepts that you can
+orchestrate together in order to produce mathematical animations: the
+**mathematical object** (or **mobject** for short) the **animation**, and the
+**scene**.  As we will see in the following sections, each of these three
+concepts is implemented in manim as a separate class: the :class:`.Mobject`,
+:class:`.Animation`, and :class:`.Scene` classes.
+
+.. note:: It is recommended that you read the tutorials :doc:`quickstart` and
+          :doc:`a_deeper_look` before reading this page.
+
+
+********
+Mobjects
+********
+
+Mobjects are the basic building block for all manim animations.  Each class
+that derives from :class:`.Mobject` represents an object that can be displayed
+on screen.  For example, simple shapes such as :class:`.Circle`,
+:class:`.Arrow`, and :class:`.Rectangle` are all mobjects.  More complicated
+constructs such as :class:`.Axes`, :class:`.FunctionGraph`, or
+:class:`.BarChart` are mobjects as well.
+
+If you try to display on screen an instance of :class:`.Mobject`, you will only
+see an empty frame.  The reason is that the :class:`.Mobject` class is an
+abstract base class of all other mobjects, i.e. it does not have any
+pre-determined visual shape that can be displayed on screen.  It is only the
+skeleton of a thing that *could* be displayed.  Therefore, you will rarely need
+to use plain instances of :class:`.Mobject`; instead you will most likely
+create instances of its derived classes.  One of these derived classes is
+:class:`.VMobject`.  The ``V`` stands for Vectorized Mobject.  In essence, a
+vmobject is a mobject that uses vector graphics to be displayed.  Most of the
```

Comment:
Maybe hyperlink to a Wikipedia article on Vector Graphics: https://en.wikipedia.org/wiki/Vector_graphics#:~:text=Vector%20graphics%20are%20computer%20graphics,form%20polygons%20and%20other%20shapes.